security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+[[security_and_privacy_of_ln]]
+== Security and Privacy of the Lightning Network
+In this chapter, we look at the issues related to the security and privacy of the LN.
+
+= Why is privacy important? =
+
+The key value proposition of cryptocurrency is censorship resistant money.
+It has been shown many times that the power of governments over money is often abused.
+Bitcoin offers individuals the possibility of storing and transferring their wealth without restrictions.
+
+Lightning continues this mission.
+It aims at providing a better user experience with instant payments while not sacrificing censorship resistance.
+However, the LN operates under a different security model, which introduces novel security and privacy challenges.
+
+= Definitions of privacy =
+
+Privacy is a complex topic.
+It is hard to precisely define what we mean by privacy.
+Therefore, the question: "Is Lightning private?" has no direct answer.
+Privacy researchers approach this issue as follows.
+First, they define a _security model_ that specifies what an adversary is capable of and aims to achieve.
+Then, they describe the relevant properties of the system and check whether it conforms to the requirements.
+A security model is based on a set of underlying _security assumptions_.
+In cryptographic systems, they are often centered around the mathematical properties of the cryptographic primitives such as ciphers, signatures, and hash functions.
+The security assumptions of the Lightning Network are that the ECDSA signatures, SHA-256 hash function, and other cryptographic functions used in the protocol behave within their security definitions.
```

Comment:
Only an intro paragraph, but I think one thing worth mentioning here is that since LN is a _networked_ protocol, any security model must also factor in how interaction with the network and also topological constraints affect the privacy of the network. As an example, since we use an onion routing protocol for HTLC extension, we by default don't assume "protection" against a global passive adversary, as such an adversary can use packet+timing information to deduce payment flows.